introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi i'm Tung, and I love to learn new things.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?